doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,241 +63,400 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
-| device_name | String | Yes | must be the device_name defined in server_information(###6.3.3), which would be set on Server from BMC|
-| node_name | String | Yes | node name for the compute to deploy, e.g: compute01 |
-| az_name | String | Yes | availability zone name which this node belongs to |
-| ha_name | List | Yes | HA name(s) which this node assigned to.|
-| vnic_type | String | Yes | VIM verification needs know compute node type: OVS,DPDK_OVS or SR-IOV node.|
-| os_reserved_cores | String | No | Designed and planned by provider itself, may be none|
-| ovs_reserved_cores | String | No | Designed and planned by provider itself, may be none |
+| APPLICATION_LAYER | String | Yes |  VIM  or storage |
+| DOMAIN | String | Yes | name of VIM/storage software product  |
+| VENDOR_NETWORK_PLANE | String | Yes | network plane designed/needed by software product |
+| NETWORK_PLANE | String | Yes | corresponding network plane in user view, like Manage, Storage, service  |
+| VENDOR | String | Yes | vendor of software product |
+| VLAN_ID | List | Yes |  designed VLAN id or id list for each network plane |
+| IP_SEGMENT | String | Yes | assigned IP segments |
+| GATEWAY | String | Yes | gateway IP for each IP range |
+| SWITCH_CONFIG_TYPE | String | Yes |   |
+
+<p align="center"><b>Table 6-3-7:</b> Network planning information.</p>
 
-<p align="center"><b>Table 6-3-7:</b> Compute information.</p>
 
 <a name="6.3.8"></a>
-### 6.3.8 Distributed storage information
-List of storage nodes that designed for current VIM deployment.
+### 6.3.8 TOR VLAN configuration information
+Multiple instances are expected, one instance for each TOR. 
 
-| Field #      | type   | mandatory | Instruction      |
-|--------------|--------|-----------|------------------|
-| cluster_name | String | Yes       | cluster name     |
-| device_infos | List   | Yes       | a list of device |
+| Field # | type | mandatory | Instruction |
+|----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| DEVICE_NAME | String | Yes |   |
+| VENDOR | String | Yes |  |
+| DEVICE_MODEL | String | Yes |  |
+| DEVICE_SN | String | Yes |  |
+| BMC_IP | String | Yes |  |
+| SSH_USER | String | Yes |  |
+| SSH_PASSWORD | String | Yes |  |
+| ENABLE_PASSWORD | String | Yes |  |
+| PORT | List | Yes |  group multiple ports with same VLAN configuration, and separate different port group with ";" |
+| VLAN_TYPE | List | Yes | tag or untag |
+| VLAN_ID | List | Yes |  group multiple VLAN with same configuration requirements, and separate different VLAN group with ";" |
+| PORT_TYPE | List | Yes | trunk or access or hybrid  |
+
+<p align="center"><b>Table 6-3-8:</b> TOR VLAN information.</p>
 
-<p align="center"><b>Table 6-3-8-1:</b> Cluster information.</p>
 
+<a name="6.3.9"></a>
+### 6.3.9 EOR VLAN configuration information
```

Comment:
EOR?

Reply:
End of Rack.
I have changed it to Aggregation Switch